device.c

```diff
@@ -83,7 +83,27 @@ static void device_lock(struct device *device)
 		err(1, "failed to lock lockfile %s", lock);
 }
 
+static bool device_check_access(struct device *device,
+				const char *username)
+{
+	struct device_user *user;
+
+	if (!device->users)
+		return true;
+
+	if (!username)
```

Comment:
Isn’t there a risk of breakage if cdba-server isn’t in sync with the cdba-shell script ?

Reply:
Default files have no users specified, so there is no breakage.
If one wants to use ACL, both cdba-shell and cdba-server must be updated.